docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
+| endpoints | regionOne                                                                              |
+|           |   admin: http://10.255.2.253:8080                                                      |
+|           | regionOne                                                                              |
+|           |   internal: http://10.255.2.253:8080/v1/AUTH_01234567890123456789012345689012          |
+|           | regionOne                                                                              |
+|           |   public: https://stack.nerc.mghpcc.org:13808/v1/AUTH_01234567890123456789012345689012 |
+|           |                                                                                        |
++-----------+----------------------------------------------------------------------------------------+
+```
+
+## Configure minio client
+
+For each environment, create a MinIO alias configuration using the base URL of the "public" interface of the object-store service and the access key and secret your generated earlier:
+
+```
+$ mc alias set kaizen https://kzn-swift.massopen.cloud 1234567890ABCDEF1234567890ABCDEF 1234567890ABCDEF1234567890ABCDEF
+$ mc alias set nerc https://stack.nerc.mghpcc.org:13808 FEDCBA0987654321FEDCBA0987654321 FEDCBA0987654321FEDCBA0987654321
+```
+
+## Create volume backup in source environment
+
+Create a volume backup in kaizen:
+
+```
+$ openstack --os-cloud kaizen volume backup create --container <bucket_name> <volume id or name>
+```
+
+In the above command line, `<bucket_name>` is the name of the object storage container that will hold your backup. This must be unique across all users, so using your username as a prefix is probably a good idea (e.g., `--container larsks-backups`).
+
+This will create a set of objects in the named bucket. You can use the MinIO client to inspect it:
+
+```
+$ mcli ls kaizen
+[2022-03-17 13:48:47 EDT]     0B larsks-backups/
+$ mcli ls kaizen/larsks-backups/
+[2022-03-21 20:02:55 EDT]     0B volume_bb2086dc-d6a4-4346-9809-f47fbde673f3/
+```
+
+## Create target container in destination environment
+
+We need to create a bucket in the target environment to receive the backup:
+
+```
+$ mcli mb nerc/larsks-backups
+```
+
+## Copy the backup
+
+Use the MinIO client to copy the objects from the MOC environment to the NERC environment:
+
+```
+$ mcli mirror kaizen/larsks-backups/volume_bb2086dc-d6a4-4346-9809-f47fbde673f3 nerc/larsks-backups/volume_bb2086dc-d6a4-4346-9809-f47fbde673f3
+```
+
+## Copy backup record from source to destination
+
+Now that we've copied the backup *data* into the NERC environment, we need to register the backup with the NERC backup service. We do this by copying metadata from Kaizen to the NERC:
+
+```
+$ openstack --os-cloud kaizen volume backup record export -f value > record.txt
+$ openstack --os-cloud nerc volume backup record import -f value $(cat record.txt)
```

Comment:
When I attempted to import I checked the values of the volume backup record to confirm that matched and they did not.
Here is the export
```
openstack --os-cloud kaizen volume backup record export -f value e8c0b62a-4067-4c39-baa6-5d3c1dcd7e0d > record.txt
ubuntu@migration-test:~$ cat record.txt
cinder.backup.drivers.swift.SwiftBackupDriver
eyJzdGF0dXMiOiAiYXZhaWxhYmxlIiwgInRlbXBfc25hcHNob3RfaWQiOiBudWxsLCAiZGlzcGxheV9uYW1lIjogbnVsbCwgImF2YWlsYWJpbGl0eV96b25lIjogIm5vdmEiLCAiZGVsZXRlZCI6IGZhbHNlLCAidm9sdW1lX2lkIjogIjU5Nzg5ZDgyLTM3ZWUtNDQ5Mi05YTUxLTVkNGI0Nzg2NGRhNCIsICJyZXN0b3JlX3ZvbHVtZV9pZCI6IG51bGwsICJ1cGRhdGVkX2F0IjogIjIwMjItMDQtMjhUMTU6MTI6MzBaIiwgImhvc3QiOiAibmV1LTMtMzktY29udHJvbDMua3puLm1vYyIsICJzbmFwc2hvdF9pZCI6IG51bGwsICJ1c2VyX2lkIjogIjJlOTQyM2JjYTlhMjRiOWY4YzZlNThmYTZlNThhZDE2IiwgInNlcnZpY2VfbWV0YWRhdGEiOiAidm9sdW1lXzU5Nzg5ZDgyLTM3ZWUtNDQ5Mi05YTUxLTVkNGI0Nzg2NGRhNC8yMDIyMDQyODE1MTAzOC9hel9ub3ZhX2JhY2t1cF9lOGMwYjYyYS00MDY3LTRjMzktYmFhNi01ZDNjMWRjZDdlMGQiLCAiaWQiOiAiZThjMGI2MmEtNDA2Ny00YzM5LWJhYTYtNWQzYzFkY2Q3ZTBkIiwgInNpemUiOiAzLCAib2JqZWN0X2NvdW50IjogNjMsICJkZWxldGVkX2F0IjogbnVsbCwgImNvbnRhaW5lciI6ICJqd2V5bC1iYWNrdXAiLCAic2VydmljZSI6ICJjaW5kZXIuYmFja3VwLmRyaXZlcnMuc3dpZnQuU3dpZnRCYWNrdXBEcml2ZXIiLCAiZHJpdmVyX2luZm8iOiB7fSwgImNyZWF0ZWRfYXQiOiAiMjAyMi0wNC0yOFQxNToxMDozNloiLCAiZGlzcGxheV9kZXNjcmlwdGlvbiI6IG51bGwsICJkYXRhX3RpbWVzdGFtcCI6ICIyMDIyLTA0LTI4VDE1OjEwOjM2WiIsICJ0ZW1wX3ZvbHVtZV9pZCI6IG51bGwsICJwYXJlbnRfaWQiOiBudWxsLCAiZW5jcnlwdGlvbl9rZXlfaWQiOiBudWxsLCAibnVtX2RlcGVuZGVudF9iYWNrdXBzIjogMCwgImZhaWxfcmVhc29uIjogbnVsbCwgInByb2plY3RfaWQiOiAiYzA4ZGViNWZiMTM1NDVmN2JhMTNjNTBjNTk1ZTg4OTUiLCAibWV0YWRhdGEiOiB7fX0=
```
here is the attempted import
```
openstack --os-cloud nerc volume backup record import -f value $(cat record.txt)
e8c0b62a-4067-4c39-baa6-5d3c1dcd7e0d
None
```
here is the check if the import succeeded, should this match the export of the other volume?
```
openstack --os-cloud nerc volume backup record export -f value e8c0b62a-4067-4c39-baa6-5d3c1dcd7e0d
cinder.backup.drivers.swift.SwiftBackupDriver
eyJkcml2ZXJfaW5mbyI6IHt9LCAiaWQiOiAiZThjMGI2MmEtNDA2Ny00YzM5LWJhYTYtNWQzYzFkY2Q3ZTBkIiwgInVzZXJfaWQiOiAiY2Q0ZjEyODMyZjhlZmNiM2NlYTAyZjE1ZDYzOWQ5MjNiNTFjY2EwNmQ0OThjNjg0MjYwODllNmQ0Y2QxZmM0NSIsICJwcm9qZWN0X2lkIjogIjBiNDA2NDA0NzU1OTQ1ZWNiYWIyMTA5ZDMyYmNlNjYxIiwgInZvbHVtZV9pZCI6ICI1OTc4OWQ4Mi0zN2VlLTQ0OTItOWE1MS01ZDRiNDc4NjRkYTQiLCAiaG9zdCI6ICJuZXJjLWN0bC0wIiwgImF2YWlsYWJpbGl0eV96b25lIjogIm5vdmEiLCAiY29udGFpbmVyIjogImp3ZXlsLWJhY2t1cCIsICJwYXJlbnRfaWQiOiBudWxsLCAic3RhdHVzIjogImF2YWlsYWJsZSIsICJmYWlsX3JlYXNvbiI6IG51bGwsICJzaXplIjogMywgImRpc3BsYXlfbmFtZSI6IG51bGwsICJkaXNwbGF5X2Rlc2NyaXB0aW9uIjogbnVsbCwgInNlcnZpY2VfbWV0YWRhdGEiOiAidm9sdW1lXzU5Nzg5ZDgyLTM3ZWUtNDQ5Mi05YTUxLTVkNGI0Nzg2NGRhNC8yMDIyMDQyODE1MTAzOC9hel9ub3ZhX2JhY2t1cF9lOGMwYjYyYS00MDY3LTRjMzktYmFhNi01ZDNjMWRjZDdlMGQiLCAic2VydmljZSI6ICJjaW5kZXIuYmFja3VwLmRyaXZlcnMuc3dpZnQuU3dpZnRCYWNrdXBEcml2ZXIiLCAib2JqZWN0X2NvdW50IjogNjMsICJ0ZW1wX3ZvbHVtZV9pZCI6IG51bGwsICJ0ZW1wX3NuYXBzaG90X2lkIjogbnVsbCwgIm51bV9kZXBlbmRlbnRfYmFja3VwcyI6IDAsICJzbmFwc2hvdF9pZCI6IG51bGwsICJkYXRhX3RpbWVzdGFtcCI6ICIyMDIyLTA0LTI4VDE1OjEwOjM2WiIsICJyZXN0b3JlX3ZvbHVtZV9pZCI6IG51bGwsICJtZXRhZGF0YSI6IHt9LCAiZW5jcnlwdGlvbl9rZXlfaWQiOiBudWxsLCAiY3JlYXRlZF9hdCI6ICIyMDIyLTA0LTI4VDE1OjEwOjM2WiIsICJ1cGRhdGVkX2F0IjogIjIwMjItMDQtMjhUMTU6NTA6MDhaIiwgImRlbGV0ZWRfYXQiOiBudWxsLCAiZGVsZXRlZCI6IGZhbHNlfQ== 
``` 
because as far as I can tell it does not.